pkg/skaffold/schema/latest/config.go

```diff
@@ -265,6 +268,26 @@ type DockerConfig struct {
 	SecretName string `yaml:"secretName,omitempty"`
 }
 
+// ResourceRequirements describes the resource requirements for the kaniko pod.
+type ResourceRequirements struct {
+	// Requests resource requests for the Kaniko pod. [More on requests](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container).
```

Comment:
nit: The link could be on `resource requests`. That makes the online doc shorter and easier to read.

Reply:
Done